assets/js/components/settings/SettingsRenderer.test.js

```diff
@@ -1,136 +0,0 @@
-/**
```

Comment:
Why was this test deleted when `SettingsRenderer` was not?

Reply:
The tests contained here were updated and moved to `SettingsActiveModule`; the tests here were testing behaviour now more specific to `SettingsActiveModule`, so it made sense to move them there.